### PR TITLE
PSAPP-233: Allow url scheme navigation within inappbrowser

### PR DESCRIPTION
**Pull request by ryanluker**
@rbierman

### CODE REVIEW

**Comment by rbierman** [comment]
Can you clean up your commit history and at least remove the WIP commit?

**Comment by ryanluker** [comment]
can do!

**Comment by ryanluker** [comment]
@rbierman Review again. I didnt do the config variable read in, but that can wait til next hipsprint

File: src/android/InAppBrowser.java [hunk 1]
```diff
@@ -755,6 +756,27 @@ public InAppBrowserClient(CordovaWebView webView, EditText mEditText) {
         }
 
         /**
+         * Allows the browser to use defined custom url scheme.
+         *
+         * @param view
+         * @param url
+         * @return
+         */
+        public boolean shouldOverrideUrlLoading (WebView view, String url){
```
Comment (freshgrade-dev) [inline src/android/InAppBrowser.java]: Space or no space that's the question

**Comment by rbierman** [comment]
:+1:

**Pull request merged**